src/Traits/CroppingImages.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+namespace TCG\Voyager\Traits;
+
+use Illuminate\Http\Request;
+use Illuminate\Database\Eloquent\Collection;
+use Illuminate\Database\Eloquent\Model;
+use Intervention\Image\Facades\Image;
+use File;
+
+trait CroppingImages
```

Comment:
I'm not sure I like the naming.  Most Laravel traits use the `-able` suffix (i.e. `Authenticatable`).  I'm open to suggestions, but `CroppingImages` seems weird.